docs/datasync/conflict-resolution.md

```diff
@@ -0,0 +1,93 @@
+---
+id: conflict-resolution
+title: Conflict Resolution strategies
+sidebar_label: Conflict Resolution strategies
+---
+
+A conflict resolution strategy is a JavaScript Object with two properties: `resolveUpdate` and `resolveDelete`, and is used to resolve conflicts during Update and Delete Mutations respectively. In order to use a specific strategy, one has to specify it in the call to `createDataSyncAPI` for every model that they want to use the strategy with:
+```typescript
+import {
+  createDataSyncAPI,
+  ClientSideWins,
+  ServerSideWins
+} from '@graphback/datasync';
+
+
+const {
+  typeDefs,
+  resolvers,
+  contextCreator
+} = createDataSyncAPI(
+  modelDefs, 
+  { 
+    db, 
+    dataSyncConflictMap: {
+      Comment: {
+        enabled: true,
+        // highlight-next-line
+        conflictResolution: ClientSideWins
+      },
+      Note: {
+        enabled: true,
+        // highlight-next-line
+        conflictResolution: ServerSideWins
+      }
+    }
+  }
+);
+```
+Graphback DataSync has two built-in conflict resolution strategies that come with it:
+
+### ClientSideWins
+
+This strategy ensures that updates always resolve to whatever the client had sent, even in the event of a conflict. If the corresponding object has been deleted in the database, it is restored when the client tries to update it. For delete conflicts, the corresponding object is always deleted, regardless of if it had been updated since the client last fetched it.
+
+
+### ServerSideWins
+
+This strategy ensures that in the event of a update conflict, the client's update will never overwrite any field that has changed since the client last fetched it. If the object has been deleted in the database, the client will be notified of it by way of a `ConflictError`. For delete conflicts, the client is informed of the conflict via a `ConflictError`.
```

Comment:
```suggestion
This strategy ensures that in the event of a update conflict, the client's update will never overwrite any field that has changed since the client last fetched it. If the object has been deleted in the database, the client will be notified of it by way of a [`ConflictError`](../api/graphback-datasync/classes/_util_.conflicterror.md). For delete conflicts, the client is informed of the conflict via a [`ConflictError`](../api/graphback-datasync/classes/_util_.conflicterror.md).
```

Reply:
Thanks! :)